app/src/main/java/com/kickstarter/ui/activities/compose/projectpage/AddOnsContainer.kt

```diff
@@ -74,10 +70,9 @@ fun AddOnsContainer(
     buttonEnabled: Boolean,
     buttonText: String,
     environment: Environment,
-    onItemAddedOrRemoved: (count: Int) -> Unit
+    onItemAddedOrRemoved: (count: Int) -> Unit,
+    itemAddOnCount: Int
 ) {
-    var addOnCount by rememberSaveable { mutableStateOf(0) }
```

Comment:
We don't want rememberSaveable, because if we navigate back and select a different reward, the addOnCount will display the same as before when it actually should be cleared.